server/block/wood_sign.go

```diff
@@ -0,0 +1,131 @@
+package block
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/block/model"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/go-gl/mathgl/mgl64"
+	"math"
+	"strings"
+)
+
+// WoodSign is block that you can write text on.
+type WoodSign struct {
+	transparent
+	bass//might be wrong
+
+
+	// Wood is the type of wood of the door. This field must have one of the values found in the material
+	// package.
+	Wood WoodType
+	// Pos is the position of the sign and the block entity
+	Pos cube.Pos
+	// Facing is the direction the sign is facing.
+	Facing cube.Direction
+	// Standing is whether the sign is on a block or against
+	Standing bool
+	// Text is an array with the text
+	Text string
+	// SignTextColor no idea what this does
+	SignTextColor int32
```

Comment:
You don't need to add fields if you don't know what they do 😉. As for its functionality, it's probably for dyeing signs by clicking them with a dye.

Reply:
We still need to figure out more thoroughly how this works and its details. We can't really keep a field lingering in the API of which we don't know how it works exactly.